src/datepicker/calendar.js

```diff
@@ -70,6 +72,7 @@ export default class Calendar extends React.Component<
     onMonthChange: () => {},
     onYearChange: () => {},
     onChange: () => {},
+    orientation: ORIENTATION.vertical,
```

Comment:
let's make the default horizontal

Reply:
FYI this would be a breaking change

Reply:
oh wow, I had no idea this is already supported https://codesandbox.io/s/datepicker-t6rh2 🤦‍♂